src/meta/src/rpc/service/hummock_service.rs

```diff
@@ -523,4 +524,30 @@ where
 
         Ok(Response::new(RwReceiverStream::new(rx)))
     }
+
+    async fn report_compaction_task(
```

Comment:
We put two types in ReportCompactionTaskRequest, ReportTask or HeartBeat.

Reply:
This confuses me, report_compaction_task is not part of the hummock service but proxy.

The way I understand the implementation,  it should be implemented using a client that is independent of the Proxy and then processed by the Proxy. Because we only need to handle a few interfaces.

Feel free to ignore me, we can add comments to it for context.